packages/@okta/vuepress-site/conductor.yml

```diff
@@ -1238,53 +1238,21 @@ redirects:
   - from: /guides/sign-into-spa/vue/user-info/index.html
     to: /docs/guides/sign-into-spa/vue/user-info/
   - from: /guides/sign-into-web-app/-/before-you-begin/index.html
```

Comment:
In theory, we should add the secondary From: redirect here (i.e. /guides/sign-into-web-app/-/before-you-begin) for all these old re-directs as they were not implemented properly originally. However, since these are old URLs and there've been no complaints//404 logs perhaps we're okay?

Reply:
I'm not sure if the old **from** was implemented incorrectly? What if the old-old structure didn't include `/docs/`?? At least that's what I'm assuming when I worked on the redirects. We can only assume that **from** is correct, and that we are controlling the **to** clause.

Reply:
Same here, this is what I assumed. We can't keep trying to second-guess fix the old redirects like this.